### PR TITLE
Add missing magic file to extension.json

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

**Comment by kghbln** [comment]
@paladox Thanks for the pull. Extension registration requires the magic file to be in the extension's root directory?

**Comment by paladox** [comment]
No, it just looked better. Since the .json files go in i18n/ and the magic file outside of it.

File: extension.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
 	"requires": {
 		"MediaWiki": ">= 1.33"
 	},
+	"ExtensionMessagesFiles": {
+		"MermaidMagic": "Mermaid.i18n.magic.php"
```
Comment (samwilson) [inline extension.json:15]: Isn't the filename `Mermaid.magic.php`?

Comment (paladox) [reply extension.json]: Yes. Thanks, done.

**Comment by Reception123** [comment]
@samwilson is this good to go now?

**Comment by samwilson** [comment]
> @samwilson is this good to go now?

Looks good to me! (But I'm not a maintainer here.)

**Comment by Reception123** [comment]
Oh, sorry. @kghbln ?

**Pull request merged**